homeassistant/components/matter/api.py

```diff
@@ -76,6 +121,8 @@ async def async_handle_failed_command_func(
             await func(hass, connection, msg, *args, **kwargs)
         except MatterError as err:
             connection.send_error(msg[ID], str(err.error_code), err.args[0])
+        except (MissingNode, ValueError) as err:
```

Comment:
I'd not catch ValueError. That would mean that the frontend tried to work on a Home Assistant device that doesn't exist. That would be a bug.

Reply:
ah yeah, makes sense :-)

Reply:
adjusted it